flexmeasures/ui/crud/assets.py

```diff
@@ -151,7 +151,7 @@ def index(self, msg=""):
             url_for("AssetAPI:index"), query={"account_id": current_user.account_id}
         )
         assets = [
-            process_internal_api_response(ad, make_obj=True)
+            process_internal_api_response(ad, int(ad["id"]), make_obj=True)
```

Comment:
The id is already in `ad` so this should not be necessary. I made a small improvement to PR #290, maybe that would remove the need you saw to do this?

Reply:
I think it does. I reverted the two affected lines in https://github.com/FlexMeasures/flexmeasures/pull/299/commits/3c607f8778d249ab64b78c3f09ea4095e7c1eeda just now.